### ISSUE TITLE
dplyr::bind_rows for character and factors

### ISSUE THREAD

**Issue by marcozanotti** [choonghyunryu/dlookr#102]
Hi, 

I am receiving a common error using dlookr functions working with category.
Here is a reprex

```
library(nycflights13)
library(dlookr)
flights |> diagnose_category()
```

Here is the error

```
Error in `dplyr::bind_rows()`:
! Can't combine `..1$levels` <character> and `..5$levels` <datetime<America/New_York>>.
```

I suppose it is due to some update within dplyr. 
Thanks

**Comment by choonghyunryu** [comment]
@marcozanotti 

Hi,

This issue has already been resolved in version 0.6.3.9003.
[diagnose_category(flights) yields error](https://github.com/choonghyunryu/dlookr/issues/100)

However, I have not submitted this version to CRAN yet.

I will submit that version soon.
If you want to fix the problem now, use the development version on github with the following command:

```
devtools::install_github("choonghyunryu/dlookr")
```

Thank you for writing an issue.

**Comment by marcozanotti** [comment]
Thank you very much for your help!
